x/leverage/keeper/borrows.go

```diff
@@ -89,6 +89,16 @@ func (k Keeper) AvailableLiquidity(ctx sdk.Context, denom string) sdkmath.Int {
 	return sdk.MaxInt(moduleBalance.Sub(reserveAmount), sdk.ZeroInt())
 }
 
+// AvailableLiquiditySubMetokenSupply gets the unreserved module balance of a given token, considering meToken supply.
+func (k Keeper) AvailableLiquiditySubMetokenSupply(ctx sdk.Context, denom string) (sdkmath.Int, error) {
```

Comment:
Ideas for this endless name?

Reply:
Might keep as-is, since it is very informative at least